packages/core/src/payment/payment-request-transformer.ts

```diff
@@ -159,6 +164,16 @@ export default class PaymentRequestTransformer {
             return { ...paymentMethod, id: CheckoutButtonMethodType.BRAINTREE_PAYPAL };
         }
 
+        if (
```

Comment:
Do we need to add this logic here? Can we do this where we build the payment request itself?

Reply:
I think, it is the best place for this logic due to the code above.

We can't move this logic to places like strategy because payment method loads with provided gateway and method on submitPayment method call. 

This code is temporary due to BigPay refactoring process. It will be removed right after paypalcommercealternativemethods will be separated from paypalcommerce gateway

Reply:
Is there a ticket for removing this temporary code here?

Reply:
Yes. Here it is: https://bigcommercecloud.atlassian.net/browse/PAYPAL-1921